cd/configs/pn-cfg.json

```diff
@@ -18,33 +18,45 @@
         }
     },
     "microservices": [
+        {
+            "name": "auth-fleet",
+            "repo-name": "pagopa/pn-auth-fleet",
+            "branch-name": "feature/PN-670",
+            "type": "lambdas",
+            "lambda-names": [
+                "pn-auth-fleet/main/apikeyAuthorizer",
```

Comment:
Non sarebbe meglio mettere solo i nomi senza _pn-auth-fleet/main/_?
Semmai mettere il branch a parte.

Reply:
Nel caso in cui abbia bisogno di fare dei branch per modificare solo la situazione di deploy, proprio come nel caso attuale, avere il branch nel nome mi da la flessibilità di utilizzare i template cloudformation di un branch e continuare ad usare il codice del branch principale.